### PR TITLE
Migrate Tabcontainer and Accordion components to Typescript

### PR DESCRIPTION
**Pull request by ManuelStardust**
- Migrate Accordion component. 
- Update Landscape, Forest HumanFootprint and PAConnectivity
- Migrate Tabcontainer component

### CODE REVIEW

File: src/pages/search/shared_components/TabContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React, { ReactElement } from "react";
+import AppBar from "@mui/material/AppBar";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Typography from "@mui/material/Typography";
+import SearchContext, { SearchContextValues } from "pages/search/SearchContext";
+
+interface Titles {
+  label: string;
+  icon?: string | ReactElement;
```
Comment (erikasv) [inline src/pages/search/shared_components/TabContainer.tsx:10]: what is the purpose of this prop being optional and allowing strings? I see the only place where this component is used it always send the icon and is an element.

File: src/pages/search/shared_components/TabContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  disabled?: boolean;
+  selected?: string;
```
Comment (erikasv) [inline src/pages/search/shared_components/TabContainer.tsx:12]: This prop isn't being used

File: src/pages/search/shared_components/TabContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    }
+    this.setState({ value });
+  };
+
+  render() {
+    const {
+      children,
+      titles,
+      tabClasses = "",
+      initialSelectedIndex = 1,
```
Comment (erikasv) [inline src/pages/search/shared_components/TabContainer.tsx:51]: this prop isn't being used here

File: src/pages/search/shared_components/TabContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+interface Props {
+  children: Array<React.ReactNode>;
+  titles: Array<Titles>;
+  tabClasses?: string;
+  initialSelectedIndex: number;
+}
+
+interface State {
+  value: number;
```
Comment (erikasv) [inline src/pages/search/shared_components/TabContainer.tsx:23]: Change this state name to `selectedIndex` or something like that, `value`is a terrible name (I know it was like that before :woman_facepalming: )

**Review by erikasv — CHANGES_REQUESTED** [review]

**Comment by erikasv** [comment]
Also please connect the corresponding issue

File: src/pages/search/shared_components/TabContainer.tsx (hunk 1 above)
Comment (ManuelStardust) [reply src/pages/search/shared_components/TabContainer.tsx]: You're right. solved.

File: src/pages/search/shared_components/TabContainer.tsx (hunk 2 above)
Comment (ManuelStardust) [reply src/pages/search/shared_components/TabContainer.tsx]: Solved

File: src/pages/search/shared_components/TabContainer.tsx (hunk 3 above)
Comment (ManuelStardust) [reply src/pages/search/shared_components/TabContainer.tsx]: Solved

File: src/pages/search/shared_components/TabContainer.tsx (hunk 4 above)
Comment (ManuelStardust) [reply src/pages/search/shared_components/TabContainer.tsx]: Done. I will keep it in mind for the future.

**Review by erikasv — APPROVED** [review]

**Pull request merged**